src/content/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Introduction to Azure monitoring integrations
+title: New Relic Azure solutions
```

Comment:
It's odd to me how short this doc is. I have a strong impulse to just combine this one with the Activate Azure integrations doc. But maybe there's a larger context I'm missing?

Reply:
Or maybe you could combine this with the other intro doc I just noticed :)

/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native/

Two intro docs is *at least* one too many :)

Reply:
You could stick all of these in a collapser at the start of the set up doc, for example

Reply:
it just seems a bit separate to me is all. you could install Azure Native without ever using the Azure infra integrations, and same with the .NET agent. Does that make sense? I thought it'd seem weird to go right into the Azure infra integrations as they seem so unconnected from other options.